lang/nl.json

```diff
@@ -14,13 +14,24 @@
   "Unmute": "Geluid aan",
   "Playback Rate": "Weergavesnelheid",
   "Subtitles": "Ondertiteling",
-  "subtitles off": "Ondertiteling uit",
-  "Captions": "Ondertiteling",
-  "captions off": "Ondertiteling uit",
+  "subtitles off": "ondertiteling uit",
```

Comment:
It's because the source text is the track's kind + ' off' in https://github.com/videojs/video.js/blob/master/src/js/control-bar/text-track-controls/off-text-track-menu-item.js#L23

I don't see a reason to not capitalise the translation if that's preferred. I'd rather capitalise the English too, but that would also require including the en translations which doesn't happen now (en.json is there as a template).